lib/org/openpsa/invoices/invoice/pdf.php

```diff
@@ -74,7 +74,6 @@ public function render_and_attach()
         if (!class_exists($client_class)) {
             throw new midcom_error('Could not find PDF renderer ' . $client_class);
         }
-
```

Comment:
There are no real changes in this file, only whitespace changes. I'd say revert it, since it only decreases the PR's readability